examples/upgrade/blue-green-route53/eks-green/main.tf

```diff
@@ -0,0 +1,319 @@
+provider "aws" {
```

Comment:
What do you think about this pattern to reduce the code duplication? The `cluster_blueprint` is the definition of your cluster with parameters exposed for what changes between blue and green clusters.

```hcl
module "blue_cluster" {
  source = "./modules/cluster_blueprint"
  
  color = "blue"
}
```

Reply:
ok let me give it a try